packages/meteor-accounts-saml/saml_rocketchat.coffee

```diff
@@ -1,59 +1,98 @@
+fs = Npm.require('fs')
+
 logger = new Logger 'steffo:meteor-accounts-saml',
 	methods:
 		updated:
 			type: 'info'
 
 RocketChat.settings.addGroup 'SAML'
 Meteor.methods
+
+  # Define configuration settings for each provider (name) in the
+  # admin SAML form.
+
 	addSamlService: (name) ->
 		RocketChat.settings.add "SAML_Custom_#{name}"                   , false                                                         , { type: 'boolean', group: 'SAML', section: name, i18nLabel: 'Accounts_OAuth_Custom_Enable'}
 		RocketChat.settings.add "SAML_Custom_#{name}_provider"          , 'openidp'                                                     , { type: 'string' , group: 'SAML', section: name, i18nLabel: 'SAML_Custom_Provider'}
 		RocketChat.settings.add "SAML_Custom_#{name}_entry_point"       , 'https://openidp.feide.no/simplesaml/saml2/idp/SSOService.php', { type: 'string' , group: 'SAML', section: name, i18nLabel: 'SAML_Custom_Entry_point'}
 		RocketChat.settings.add "SAML_Custom_#{name}_issuer"            , 'https://rocket.chat/'                                        , { type: 'string' , group: 'SAML', section: name, i18nLabel: 'SAML_Custom_Issuer'}
 		RocketChat.settings.add "SAML_Custom_#{name}_cert"              , ''                                                            , { type: 'string' , group: 'SAML', section: name, i18nLabel: 'SAML_Custom_Cert'}
+		RocketChat.settings.add "SAML_Custom_#{name}_public_cert_file_path"  , 'filename.crt'                                           , { type: 'string' , group: 'SAML', section: name, i18nLabel: 'SAML_Custom_Public_Cert_File_Path'}
+		RocketChat.settings.add "SAML_Custom_#{name}_private_key_file_path"  , 'filename.key'                                           , { type: 'string' , group: 'SAML', section: name, i18nLabel: 'SAML_Custom_Private_Key_File_Path'}
```

Comment:
Hi @rodrigok , I need them to show in the form.  I'm assuming that's how to do it?

The important thing is the private key so that rocketchat can sign requests to the idp.
I think the original saml package (this code is based) on also checks for the crt (it checks for `self.options.privateCert`) so that's also included.

The other thing is the example value in those configs should be `'/path/to/filename.{crt,key}'`.

If the idp doesn't verify signature when rocketchat redirects to the logout, then these 2 fields can be left blank (just tested this behaviour, turning `'redirect.validate' => FALSE` in simplesamlphp metadata sp-remote config file).